src/App.vue

```diff
@@ -751,7 +751,8 @@ export default class App extends Mixins(
         applicationDate: this.dateToYyyyMmDd(this.apiToDate(header.date)),
         legalFilings: [header.name],
         order: data.courtOrder
-      }
+      },
+      latestReviewComment: header.latestReviewComment
```

Comment:
New property from API that we need for the filing body in case the application is rejected, ie:

![image](https://github.com/user-attachments/assets/ba87e6f3-4b1f-4e17-a769-2c6230ba3e30)